doc/specs/stdlib_io.md

```diff
@@ -131,3 +131,130 @@ program demo_savetxt
     call savetxt('example.dat', x) 
 end program demo_savetxt
 ```
+
+## `disp` - display your data
+
+### Status
+
+Experimental
+
+### Class
+
+Impure subroutine.
+
+### Description
+
+Outputs a `logical/integer/real/complex/character/string_type` scalar or `logical/integer/real/complex` and rank-1/rank-2 array to the screen or a file `unit`.
+
+#### More details
+
+```fortran
+call disp( A(i, j, 2, :, 1:10) [, header, unit, brief] )    !! `i, j, ...` can be determined by `do` loop.
+```
+
+For `complex` type, the output format is `*(A25, 1X)`; 
+For    other types, the output format is `*(A12, 1X)`.
+
+To prevent users from accidentally passing large-length arrays to `disp`, causing unnecessary io blockage:
+1. If the `brief` argument is not specified, `disp` will print **the brief array content with a length of `10*50` by default**.
+2. Specify `brief=.true.`, `disp` will print **the brief array content with a length of `5*5`**;
+3. Specify `brief=.false.`, `disp` will print **`all` the contents of the array**.
```

Comment:
So if `brief` is not provided (i.e. the default), then it is neither `.true.` nor `.false.`, right?
This sounds a bit counter-intuitive to me.

Reply:
`brief` is `.false.` now.